admin/grants.sql

```diff
@@ -638,4 +638,24 @@ GRANT SELECT, INSERT
 GRANT ROLE ACCOUNTADMIN
 	TO USER ADMIN_SERVICE;
 GRANT ROLE DATA_ENGINEER 
-	TO USER DEVELOPER_SERVICE;
\ No newline at end of file
+	TO USER DEVELOPER_SERVICE;
+
+-- Allow the proxy admins to run serverless tasks
+USE ROLE ACCOUNTADMIN;
+GRANT EXECUTE MANAGED TASK
+	ON ACCOUNT
+	TO ROLE SYNAPSE_DATA_WAREHOUSE_DEV_ALL_ADMIN;
+GRANT EXECUTE MANAGED TASK
+	ON ACCOUNT
+	TO ROLE SYNAPSE_DATA_WAREHOUSE_ALL_ADMIN;
+USE ROLE SECURITYADMIN;
```

Comment:
Previous grants to `COMPUTE_XSMALL` were granted by `SYSADMIN` - are we sure we're not supposed to switch to `SYSADMIN` before running the grants below?

<img width="474" alt="image" src="https://github.com/user-attachments/assets/390b825f-330d-461c-b9ba-2c04e620cf2c" />